crypto/ec/ecp_s390x_nistp.c

```diff
@@ -0,0 +1,394 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <openssl/err.h>
+#include <openssl/rand.h>
+#include "ec_lcl.h"
+#include "s390x_arch.h"
+
+/* Size of parameter blocks */
+#define S390X_SIZE_PARAM		4096
+
+/* Size of fields in parameter blocks */
+#define S390X_SIZE_P256			32
+#define S390X_SIZE_P384			48
+#define S390X_SIZE_P521			80
+
+/* Offsets of fields in PCC parameter blocks */
+#define S390X_OFF_RES_X(n)		(0 * n)
+#define S390X_OFF_RES_Y(n)		(1 * n)
+#define S390X_OFF_SRC_X(n)		(2 * n)
+#define S390X_OFF_SRC_Y(n)		(3 * n)
+#define S390X_OFF_SCALAR(n)		(4 * n)
+
+/* Offsets of fields in KDSA parameter blocks */
+#define S390X_OFF_R(n)			(0 * n)
+#define S390X_OFF_S(n)			(1 * n)
+#define S390X_OFF_H(n)			(2 * n)
+#define S390X_OFF_K(n)			(3 * n)
+#define S390X_OFF_X(n)			(3 * n)
+#define S390X_OFF_RN(n)			(4 * n)
```

Comment:
NIT: indent

Reply:
whats wrong with indent ?

Reply:
Line 38 looks like it has extra spaces (is it tabbed or is it just how it appears in review)

Reply:
its tabbed

Reply:
Tabs are not supposed to be used according to the coding style doc.

Reply:
I thought this only applies to indent at the beginning of a line. Here is the only place docs mentions tabs:
```
Indentation is four space characters. Do not use the tab character.
```

If "tabs not to be used" applies in general, it would have to be replaced in e_aes.c s390x aes code because i used tabs there with macros (especially line breaks), too,

grep -Pr "\t" . shows tabs all over the place in platform specific asm code.

Reply:
We have made an informal exception on the perlasm code, so far.  I think that's to be revisited at some point, but now's not the time.

The current coding style is primarly for C code, and the "no tabs" rule is general, not just about indentation.  This is a C source file, so needs to be corrected accordingly.

Reply:
Looks like commit dd6b270618b8f43009999e45ad6dd03ca50bbe54 fixed the tabs in my precious submissions.

So i ll fix it for this one together with the rebase.